### DISCUSSION TITLE
How to Visualize PNG With Transparent Background?

### DISCUSSION THREAD

**Discussion by tempaccountforissue** [isl-org/Open3D#6070] [Q&A] [answered]
This is the code I'm using to make my PNG:

`vis = o3d.visualization.Visualizer()
vis.create_window('Open3D', 1080, 1080, visible=False)
vis.add_geometry(voxel_grid)
vis.update_geometry(voxel_grid)
vis.poll_events()
vis.update_renderer()
vis.capture_screen_image('C:/Users/Name/Documents/PNGs/filename.png')
vis.destroy_window()`

This produces a PNG with a white background. How can I make it produce a PNG with a transparent background?

**Comment by ssheorey** [comment, marked as answer]
Hi @tempaccountforissue Open3D doesn't support transparency in the saved image. But a workaround would be to use a "green screen" type color (something absent in your scene - white may work as well), capture the image and then post process it with imagemagick to replace the green screen with transparency. See here for help:

https://stackoverflow.com/questions/9155377/set-transparent-background-using-imagemagick-and-commandline-prompt
